src/minefield/create_test_game.cpp

```diff
@@ -0,0 +1,19 @@
+#include <minefield/create_test_game.h>
+
+//helper function for quickly instantiating games without user input
+State createTestGame(GameContext &context, unsigned int humanPlayers, unsigned int botPlayers, unsigned int width, unsigned int height, int initialMines) {
```

Comment:
#nice helper!

#could set default values for arguments so you don't have to be explicit about them while writing tests that do not require specific values